frontend/src/constants/routes.js

```diff
@@ -159,3 +160,19 @@ export const SEARCH_RESULTS = {
   dynamicRoute: ({ q, t }) => (t ? `/search?q=${q}&t=${t}` : `/search?q=${q}`),
   component: SearchResults,
 };
+
+// export const NOTICE_OF_WORK_APPLICATIONS = {
+//   route: "/dashboard/notice-of-work",
+//   dynamicRoute: ({ page, per_page, ...params }) =>
+//     `/dashboard/notice-of-work?${queryString.stringify(
+//       { page, per_page, ...params },
+//       { sort: false }
+//     )}`,
+//   component: COMPONENT_NAME_GOES_HERE,
+// };
```

Comment:
Remove? If not, perhaps you could add a TODO to explain why it's here but commented out

Reply:
ah, yes forgot about this, removing